components/net_traits/request.rs

```diff
@@ -254,6 +254,7 @@ pub struct RequestBuilder {
     pub initiator: Initiator,
     pub https_state: HttpsState,
     pub response_tainting: ResponseTainting,
+    pub crash: Option<String>,
```

Comment:
It might be nice to put a little docstring here explaining what's up with this new member. Maybe something along the lines of:

```rust
/// When this member is non-None it means that the LoadData is meant to trigger an error page showing details about a crash.
```

Reply:
done in c1a6558ca29daeb81020934a828e6cf0014f99f8